README.md

```diff
@@ -526,15 +526,44 @@ enum ImageOrientation {
 
 | Param | Type |
 | --- | --- |
-| environment | Environment | 
-| [venueApiUrl] | string | 
+| environment | Environment |
+| [venueApiUrl] | string |
 
 * getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
+* getSeatAttributesBySeatId(venueId, seatIdCollection) ⇒ Array&lt;SeatAttributes&gt;
+* getDetails(venueId, seatIdCollection) ⇒ Array&lt;VenueDetails&gt;
+* getChartDetails(productId, date) ⇒ Array&lt;ChartDetails&gt;
 
 #### ➥ getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
 *Get list of seat attributes*
-**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models* 
+**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models*
+
+| Param | Type |
+| --- | --- |
+| venueId | string |
+
+#### ➥ getSeatAttributesBySeatId(venueId, seatIdCollection) ⇒ Array&lt;SeatAttributes&gt;
+*Get list of seat attributes by seat id*
+**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models*
+
+| Param | Type |
+| --- | --- |
+| venueId | string |
+| seatIdCollection | array |
```

Comment:
Specify type of items expected in this array

Reply:
done